op-chain-ops/genesis/setters.go

```diff
@@ -29,23 +29,22 @@ func SetProxies(hh *hardhat.Hardhat, db vm.StateDB) error {
 	if err != nil {
 		return err
 	}
-	proxyAdmin, err := hh.GetDeployment("ProxyAdmin")
-	if err != nil {
-		return err
-	}
+	proxyAdminSlot := predeploys.ProxyAdminAddr.Hash()
 
-	for i := uint64(0); i <= 2048; i++ {
+	for i := uint64(0); i < 2048; i++ {
```

Comment:
Where does 2048 come from?

Reply:
Random number that @mslipper chose that is large enough to allow for many predeploys but small enough to make the genesis file not be massive